src/lib/util/dict_tokenize.c

```diff
@@ -437,12 +437,12 @@ static int dict_process_type_field(dict_tokenize_ctx_t *dctx, char const *name,
 	if (fr_type_is_null(type)) {
 		fr_dict_attr_t *da = *da_p;
```

Comment:
You can nuke this line, too.  Otherwise the compiler will complain about unused variables